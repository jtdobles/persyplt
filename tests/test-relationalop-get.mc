void main(){
	int a = 3;
	if(a >= -2){
		prints('a is greater or equal to -2');
	}else{
		prints('a is not greater or equal to -2');
	}
	if(a >= 3){
		prints('a is greater or equal to 3');
	}else{
		prints('a is not greater or equal to 3');
	}
	if(a >= 5){
		prints('a is greater or equal to 5');
	}else{
		prints('a is not greater or equal to 5');
	}
}