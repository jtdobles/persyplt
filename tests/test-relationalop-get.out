'a is greater or equal to -2'
'a is greater or equal to 3'
'a is not greater or equal to 5'
